test/e2e/network/service.go

```diff
@@ -2634,6 +2634,20 @@ var _ = SIGDescribe("Services", func() {
 		framework.ExpectNoError(err)
 		e2eservice.WaitForServiceUpdatedWithFinalizer(cs, svc.Namespace, svc.Name, true)
 	})
+
+	ginkgo.It("should find a service from listing all namespaces", func() {
+		ginkgo.By("fetching services")
+		svcs, _ := f.ClientSet.CoreV1().Services("").List(metav1.ListOptions{})
+
+		foundSvc := false
+		for _, svc := range svcs.Items {
+			if svc.ObjectMeta.Name == "kubernetes" {
+				foundSvc = true
+			}
+		}
+
+		framework.ExpectEqual(foundSvc, true, "could not find service in service list in all namespaces")
```

Comment:
Make the log more explicit? e.g. `could not find kubernetes service ...`

Reply:
Updated in 9d8bcef56ba